README.md

```diff
@@ -32,28 +34,71 @@ Google Earth Engine requires a **Google Cloud Project**. To find your Project ID
 - Select the sign-in under `Plugins > Google Earth Engine > Sign-In`.
 - Follow prompts to enter your project ID.
 
-### 4️⃣ Use Earth Engine Directly in QGIS 🎉
+---
+
+## 🌍 Using Earth Engine in QGIS
 
-Once the plugin is set up, you can start using Earth Engine within QGIS! Try adding a dataset:
+### 1. Code Usage
```

Comment:
I don't know if these should be numbered. Above in the README, we number steps because they should be done in order.  In this case, they're independent of eachother.